src/dotnet/APIView/APIViewWeb/Helpers/CosmosQueryHelpers.cs

```diff
@@ -0,0 +1,30 @@
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace APIViewWeb.Helpers
+{
+    public class CosmosQueryHelpers
+    {
+        public static string ArrayToQueryString<T>(IEnumerable<T> items)
+        {
+            var result = new StringBuilder();
+            result.Append("(");
+            foreach (var item in items)
+            {
+                if (item is int)
+                {
+                    result.Append($"{item},");
+                }
+                else
+                {
+                    result.Append($"\"{item}\",");
+                }
+
+            }
+            result.Remove(result.Length - 1, 1);
```

Comment:
Do we need to worry about a case where items is empty? It's better to handle it.